docs/configuring-playbook-backup-borg.md

```diff
@@ -0,0 +1,52 @@
+# Setting up borg backup (optional)
+
+The playbook can install and configure [borgbackup](https://www.borgbackup.org/) with [borgmatic](https://torsion.org/borgmatic/) for you.
+The backup will run based on `matrix_backup_borg_schedule` var (systemd timer calendar), default: 4am every day
+
+## Prerequisites
+
+1. Create ssh key:
+
+```bash
+ssh-keygen -t ed25519 -N '' -C matrix
+```
+
+2. Add public part of that ssh key to your borg provider / server:
+
+```bash
+# example to append the new PUBKEY contents, where:
+# PUBKEY is path to the public key,
+# USER is a ssh user on a provider / server
+# HOST is a ssh host of a provider / server
+cat PUBKEY | ssh USER@HOST 'dd of=.ssh/authorized_keys oflag=append conv=notrunc'
+```
+
+## Adjusting the playbook configuration
+
+Minimal working configuration (`inventory/host_vars/matrix.DOMAIN/vars.yml`) to enable borg backup:
+
+```yaml
+matrix_backup_borg_enabled: true
+matrix_backup_borg_repository: "USER@HOST:REPO"
+matrix_backup_borg_passphrase: "PASSPHRASE"
+matrix_backup_borg_ssh_key: |
```

Comment:
Should we put `_private` in the name somewhere, to make it more explicit that this is the private key?

Should we have a variable for the public key as well? If not for anything else, than at least for keeping a backup as part of `vars.yml`?

Reply:
Only private key is needed and I don't thing we need to add a public key var for backup purposes - you can restore public part from the private, IIRC